autometacal/python/datasets/CFIS.py

```diff
@@ -0,0 +1,150 @@
+""" TensorFlow Dataset of simulated CFIS images."""
+import tensorflow_datasets as tfds
+import tensorflow as tf
+import numpy as np
+import galsim as gs
+
+_CITATION = """
+"""
+
+_DESCRIPTION = """
+"""
+
+class CFISConfig(tfds.core.BuilderConfig):
+  """BuilderConfig for CFIS Galaxies."""
+
+  def __init__(self, *, galaxy_type="parametric", 
+                data_set_size=1000, stamp_size=51,
+                shear_g1=0.0, shear_g2=0.0, **kwargs):
+    """BuilderConfig for CFIS.
+    Args:
+      size: size of sample.
+      stamp_size: image stamp size in pixels.
+      pixel_scale: pixel scale of stamps in arcsec.
+      **kwargs: keyword arguments forwarded to super.
+    """
+    v1 = tfds.core.Version("0.0.1")
+    super(CFISConfig, self).__init__(
+        description=("Galaxy stamps"),
+        version=v1,
+        **kwargs)
+      
+    # Adjustable parameters
+    self.galaxy_type = galaxy_type
+    self.data_set_size = data_set_size
+    self.stamp_size = stamp_size
+    self.kstamp_size = stamp_size
+    self.shear_g1 = shear_g1
+    self.shear_g2 = shear_g2
+
+    # Fixed parameters
+    self.pixel_scale = 0.187
+    self.psf_fwhm = 0.65     # TODO: add varying PSF
+    self.psf_e1 = 0.0
+    self.psf_e2 = 0.025
+
+
+class CFIS(tfds.core.GeneratorBasedBuilder):
+  """DatasetBuilder for simulated CFIS dataset."""
+
+  VERSION = tfds.core.Version('0.0.1')
+  RELEASE_NOTES = {
+      '0.0.1': 'Initial release.',
+  }
+  
+  BUILDER_CONFIGS = [CFISConfig(name="parametric_1k", galaxy_type="parametric", data_set_size=1000),
+                     CFISConfig(name="parametric_shear_1k", galaxy_type="parametric", data_set_size=1000, shear_g1=0.02)]
+
+  def _info(self) -> tfds.core.DatasetInfo:
+    """Returns the dataset metadata."""
+    # TODO: Specifies the tfds.core.DatasetInfo object
+    return tfds.core.DatasetInfo(
+        builder=self,
+        description=_DESCRIPTION,
+        features=tfds.features.FeaturesDict({
+          'obs': tfds.features.Tensor(shape=[self.builder_config.stamp_size,
+                                                   self.builder_config.stamp_size],
+                                        dtype=tf.float32),
+          'psf': tfds.features.Tensor(shape=[self.builder_config.stamp_size,
+                                                   self.builder_config.stamp_size],
+                                        dtype=tf.float32),    
+          # 'gal_kimage': tfds.features.Tensor(shape=[2, self.builder_config.kstamp_size,
```

Comment:
what do you have these commented lines? If they are not needed we should remove to avoid cluttering the code

Reply:
that;s a good point, I'll rmove this stuff for now